packages/cozy-client/src/links/DataProxyLink.js

```diff
@@ -23,24 +30,80 @@ export default class DataProxyLink extends CozyLink {
    */
   registerDataProxy(dataproxy) {
     this.dataproxy = dataproxy
+    this._flushQueue()
```

Comment:
should we await?

Reply:
If we handle correctly the catch, yes, that's true. But I think that at the moment, this is not the case.

Reply:
It is catch in  client.query : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L1059